Chinese/大话西游.md

```diff
@@ -144,4 +144,6 @@
 
 [庐山恋](lushan/lushan_love_story.md)
 
+[人工智能](ryanng/zhizunbo.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.